meta/devices/friendlyarm_nanopi-r1.yaml

```diff
@@ -0,0 +1,43 @@
+vendor: FriendlyARM
+model: NanoPi R1
+bootloader: unknown
+cpu:
+  model: Allwinner H3
+  cores: 4
+  mhz: 1200
+memory_mb:
+  - 512
+  - 1024
+storage:
+  - type: microSD
+device_type: WiFi Router
+usb:
+  - type: A
+    ports: 2
```

Comment:
I'm not sure, but maybe num_ports would be better here for clarity (same for ethernet).

Reply:
currently I suffixed variable names with their type, would you be okay with `port_cnt`?

Reply:
Apart from that, we validation schema can also contain titles and descriptions which can explain variables in detail

Reply:
I don't like "cnt", I'd use either port_count or port_num then.

Reply:
ack

Reply:
It's now `port_count`.